ws-nextjs-app/pages/ws/languages.page.tsx

```diff
@@ -0,0 +1,529 @@
+
+// @ts-ignore
+const languageToggle = e => {
+  e.stopPropagation();
+  const elements = document.querySelectorAll('.panel, .toggle');
+  for (let i = 0; i < elements.length; ++i) {
+    const classNames = elements[i].className.split(' ');
+    const indexOfHide = classNames.indexOf('hide');
+    if (indexOfHide !== -1) {
+      classNames.splice(indexOfHide, 1);
+    } else {
+      classNames.push('hide');
+    }
+    elements[i].className = classNames.join(' ');
+  }
+};
+
+const pageTitle = 'News in your language - BBC News';
+const pageDescription = 'A list of BBC World Service language services';
+const lang = 'en';
+
+const atiAnalytics = {
+  campaigns: null,
+  categoryName: null,
+  contentType: STATIC_PAGE,
+  language: lang,
+  ldpThingIds: null,
+  ldpThingLabels: null,
+  pageIdentifier: 'ws.languages.page',
+  pageTitle,
+  producerId: null,
+  producerName: 'DEFAULT',
+};
+
+const pageLayout = () => {
+  return (
+    <>
+      <ATIAnalytics atiData={atiAnalytics} />
+      <ChartbeatAnalytics title={pageTitle} />
+      <MetadataContainer
+        title={pageTitle}
+        lang={lang}
+        description={pageDescription}
+        openGraphType="website"
+        hasAmpPage={false}
+      />
+      <style>{morphCSS1}</style>
+      <style>{morphCSS2}</style>
+      <main>
+        <div id="page" className="atlas-languages-page">
+          <div className="container c-open" role="main">
+            <div className="column-clearfix">
+              <div className="column--primary">
+                <h1 className="story-body__h1">
+                  Get the news in your language
+                </h1>
+                <p className="language-switcher">
+                  <a
```

Comment:
Can this be a `button` rather than an anchor if we're using it like a button?

Reply:
yeah, definitely _should be_

I just put what was there already.

Reply:
I'll try and put a fix out